packages/ui/src/domainModules/collectionMammals/components/MammalManager/MainColumn/RecordForm/unitSpecs/collectingLocation/index.js

```diff
@@ -36,8 +37,8 @@ const parts = [
   {
     componentName: 'Input',
     componentProps: {
-      columnProps: { width: 7 },
-      fluid: true,
+      columnProps: { width: 6 },
+      style: { width: '70%' },
```

Comment:
should not set width like this, then there's no point with the column width. if you want it to be in a new row, below the previous column, pass `clear: true` into columnProps

Reply:
Lisa wants to have width 3, but it won't work, and in order to make it not in same line with Higher geography, I had to make it 6, and add 70% to reduce the width.

Reply:
Yes but have you tried this thing I wrote in the comment?

> if you want it to be in a new row, below the previous column, pass clear: true into columnProps